### PR TITLE
Add crony time synchronization configuration in the ignition files

### PR DESCRIPTION
**Pull request by schabrolles**
We were faced to issues when trying to install OCP on systems where firmware clock was not properly configured.

The issue was that the bastion/helper node (based on rhel) add chrony/ntp configured (which resync time) but cores node were not. 
All certificates generated by bastion cannot be used by nodes during installation (1h time difference).

We think it could be a good option to add the possibility to specify the time-server to add to the coreos for chrony configuration.

```
chronyconfig:
  enabled: true
  content:
    - server: ntp1.example.com
      options: iburst
    - server: ntp2.example.com
      options: iburst
```

### CODE REVIEW

**Comment by yussufsh** [comment]
@schabrolles Thanks for the PR.
Could you please:
1. Create a issue to discuss what all changes you want to propose.
2. Mention the issue# in the commit message. See [here](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/CONTRIBUTING.md#pull-request-process).
3, Rebase/squash your commits.

File: playbooks/roles/ocp-config/tasks/main.yaml [hunk 1]
```diff
@@ -101,6 +101,28 @@
     line: '\1mastersSchedulable: False'
     backrefs: yes
 
+- name: Chrony configuration
+  block:
```
Comment (yussufsh) [inline playbooks/roles/ocp-config/tasks/main.yaml:105]: Suggestion would be to move full bock to a chrony.yaml task file. See squid.yaml for example.
Also, please mention the new variable in the role README.

**Comment by schabrolles** [comment]
> @schabrolles Thanks for the PR.
> Could you please:
> 
> 1. Create a issue to discuss what all changes you want to propose.
> 2. Mention the issue# in the commit message. See [here](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/CONTRIBUTING.md#pull-request-process).
>    3, Rebase/squash your commits.

Do I have do create a new PR with Rebase/squash your commits ?

**Comment by schabrolles** [comment]
This one could be closed as it is replaced by #34

**Pull request closed without merging**